modules/survey_accounts/test/TestPlan.md

```diff
@@ -14,27 +14,29 @@
     Make sure that if you don’t fill out a field that is required,
     and you hit save and continue fields you have filled out are not erased.
     [Manual Testing]
-6.  Add Survey button:
-    a. Try sending the URL to yourself -  make sure that both email addresses match in order for the “Email Survey” button to work
-      (try mismatched email addresses to make sure that the "Email Survey" button remains inactive).
-    b. Once you hit “Email Survey” you should get a blank page where you can customize an email to go along with the URL –
+6.  Add Survey button: 
```

Comment:
can you fix the extra whitespace at the end of these lines that github is showing in the diff?